notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1566 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ! gsutil rm -r $BUCKET_URI
</pre><p>Put this in an <code>if statement</code>to prevent inadvertent destruction of someone's personal bucket.</p><p> </p><pre class="ql-syntax" spellcheck="false">DELETE_BUCKET = True

if DELETE_BUCKET:
	...&nbsp;
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/882/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='83'/>